zulipterminal/cli/run.py

```diff
@@ -93,7 +93,7 @@ def fetch_zuliprc(zuliprc_path: str) -> None:
 
     while res.status_code != 200:
         print(in_color('red', "\nUsername or Password Incorrect!\n"))
-        res = get_api_key(realm_url)
```

Comment:
Huh, how does this line work without the fix when it is being called at L94?

Reply:
I think this is a bug.